frontend/src/views/DashboardPage/components/SecretInputRow/SecretInputRow.tsx

```diff
@@ -245,6 +254,9 @@ export const SecretInputRow = memo(
                         isKeySubDisabled.current = false;
                         field.onBlur();
                       }}
+                      onChange={(val)=>{
+                        onChange(hasEqualSign(val.target.value)?splitKeyValue(val.target.value):val);
```

Comment:
The issue with this logic is whenever secret value has = it gets split into two parts which can confuse user. 

This should happen only when user is pasting into the input field.

Reply:
Also are you sure value is getting saved, just asking to be sure. Coz the editorRef change may be affected only in ui not in state